### PR TITLE
Display stream concentration values in Water Quality analysis tab

### PR DESCRIPTION
**Pull request by kellyi**
This PR updates the Water Quality analysis tab and popup to display catchment values for the new concentration columns added in #1531. The output currently looks like this:

screen shot 2016-10-12 at 5 28 41 pm

& ->

screen shot 2016-10-12 at 5 23 27 pm

The first commit here is a shim css change to adjust the tab width so that the water quality tab will display. The second adjusts `calcs.py` to retrieve the new columns from the `drb_catchment_water_quality` table and return them with the analysis results, and updates the frontend ui to add logic and templates for displaying the new columns in the table added in #1526 and a utility function to return the average of the concentration values, skipping ones which were `null`.
 
**Testing**
- get this branch, `vagrant up`, `vagrant ssh app` then `cd /vagrant/scripts/aws` and run `./scripts/setupdb.sh -q` to get the updated water quality table
- visit `localhost:8000` in the browser and open the browser console
- select an AOI within the DRB. When the analysis results return verify that you see the new columns in the water quality tab and that when you click on one of the rows' ids, the new values are also visible in the popup.

Connects #1532

### CODE REVIEW

**Comment by kellyi** [comment]
Rebasing this on develop; will push updated commits momentarily.

**Comment by kellyi** [comment]
Made a typo in one of the templates, which was causing the tests to fail. Pushed an amended commit.

**Comment by kellyi** [comment]
@azavea-bot rebuild

File: src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html [hunk 1]
```diff
@@ -12,17 +12,43 @@
             <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
                 Total TSS (kg/ha)
             </th>
+            <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
+                Avg TN (mg/l)
+            </th>
+            <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
+                Avg TN (mg/l)
```
Comment (kellyi) [inline src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html]: Yeah! I thought I had pushed it but it looks like I did not. commit ff054cb should have it.

**Comment by kellyi** [comment]
@azavea-bot rebuild

File: src/mmw/apps/modeling/calcs.py [hunk 2]
```diff
@@ -107,10 +107,10 @@ def catchment_water_quality(geojson):
...
     sql = '''
           SELECT nord, areaha, tn_tot_kgy, tp_tot_kgy, tss_tot_kg,
-              tn_urban_k, tn_riparia, tn_ag_kgyr, tn_natural, tn_pt_kgyr,
-              tp_urban_k, tp_riparia, tp_ag_kgyr, tp_natural, tp_pt_kgyr,
-              tss_urban_, tss_rip_kg, tss_ag_kgy, tss_natura,
-              ST_AsGeoJSON(geom) as geom
+          tn_urban_k, tn_riparia, tn_ag_kgyr, tn_natural, tn_pt_kgyr,
+          tp_urban_k, tp_riparia, tp_ag_kgyr, tp_natural, tp_pt_kgyr,
+          tss_urban_, tss_rip_kg, tss_ag_kgy, tss_natura,
+          ST_AsGeoJSON(geom) as geom, tn_yr_avg_, tp_yr_avg_, tss_concmg
```
Comment (arottersman) [inline src/mmw/apps/modeling/calcs.py]: Consider moving these before the `geom` column

**Comment by arottersman** [comment]
Tested! The new columns are there, the new row is in the popup, and the tests pass. :) +1

**Comment by kellyi** [comment]
Thanks for taking a look at this! Going to merge this once the tests pass.

**Pull request merged**